node/tools/Makefile

```diff
@@ -1,12 +1,34 @@
 .PHONY: run-node node-configs clean
 IP?=127.0.0.1:3054
+DOCKER_IP=172.12.0.10
 
-run-node:
+node:
 	export RUST_LOG=INFO && cd node-configs/${IP} && cargo run -- --database ./database/${IP} 
 
-node-configs:
+node_configs:
 	cargo run --bin localnet_config -- --input-addrs addresses.txt --output-dir node-configs
 
-clean:
+node_docker:
+	rm -rf addresses.txt
```

Comment:
can we make Makefile do not touch files in the source directory (do this stuff in /tmp or sth, I don't use makefiles much)? For example node_docker command overwrites the addresses.txt file that was required to be provided in the local node setup according to README.txt and I think it is not nice.

Reply:
Good point. I've added another directory with the config specifically for the nodes running in Docker. This way, Docker will only use this config, and the original `addresses.txt` will remain untouched.